package.json

```diff
@@ -36,9 +36,12 @@
   "devDependencies": {
     "babel": "^5.2.16",
     "babel-core": "^5.2.6",
+    "babel-eslint": "^4.1.3",
     "babel-loader": "^5.0.0",
     "babel-plugin-object-assign": "^1.0.1",
     "brfs": "^1.2.0",
+    "eslint": "^1.7.3",
```

Comment:
A lot of devs will have their editor set up to do linting automatically. Mine, for example, sees an `.eslintrc` in the root and then looks for a local `eslint` binary. This makes it easier for me to switch between projects that use `eslint` or `jshint`.